tools/topology/topology2/include/common/output_audio_format.conf

```diff
@@ -1,7 +1,7 @@
-# Class definition for spirce pin audio format object
```

Comment:
I like the spirce pin. That sounds like a good concept that's not overloaded just yet.